project-base/app/config/directories.yml

```diff
@@ -0,0 +1,15 @@
+parameters:
```

Comment:
Are here some directories that have to be always created, or doesn't make sense for a programmer to change them?
In that case, I would like to see some default configuration in framework, so upgrading will be much easier.